### PR TITLE
Feature/2291 cash contributions question translations

### PR DESCRIPTION
**Pull request by paulmsmith**
## Proposed changes

[Ticket detailed here](https://trello.com/c/gew2oCo5/2291-abstract-strings-from-cash-contributions-question-page)

### CODE REVIEW

File: config/locales/funding_application/gp_project/cash_contributions/question/cy.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+cy:
+  application:
+    cash_contributions_question:
+      page_title: "Are you getting any cash contributions to your project?"
+      page_heading: "Are you getting any cash contributions to your project?"
```
Comment (ptrelease) [inline config/locales/funding_application/gp_project/cash_contributions/question/cy.yml:5]: No 'any' in the original text.

Comment (ptrelease) [reply config/locales/funding_application/gp_project/cash_contributions/question/cy.yml:5]: the heading looks good though

File: config/locales/funding_application/gp_project/cash_contributions/en.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+en-GB:
+  application:
+    cash_contributions:
+      page_title: "Cash contributions"
+      page_heading: "Cash contributions"
+      headings:
+        1: "Your cash contributions"
+        2: "Add a cash contribution"
+        3: "You have not added any cash contributions yet"
```
Comment (ptrelease) [inline config/locales/funding_application/gp_project/cash_contributions/en.yml:9]: In a separate OR too - but we can lose the 'yet'

File: app/views/funding_application/gp_project/cash_contributions/show.html.erb [hunk 3]
```diff
@@ -65,14 +71,15 @@
           <tr class="govuk-table__row">
 
             <th scope="col" class="govuk-table__header govuk-!-width-one-half">
-              Description
+              <%= t('application.cash_contributions.headings.1') %>
```
Comment (ptrelease) [inline app/views/funding_application/gp_project/cash_contributions/show.html.erb:74]: should be th section

File: app/views/funding_application/gp_project/cash_contributions/show.html.erb [hunk 4]
```diff
@@ -65,14 +71,15 @@
...
-            <th scope="col" class="govuk-table__header">Secured</th>
+            <th scope="col" class="govuk-table__header">
+              <%= t('application.cash_contributions.headings.2') %>
+            </th>
 
-            <th scope="col"
-                class="govuk-table__header govuk-table__header--numeric">
-              Value
+            <th scope="col" class="govuk-table__header govuk-table__header--numeric">
+              <%= t('application.cash_contributions.headings.3') %>
```
Comment (ptrelease) [inline app/views/funding_application/gp_project/cash_contributions/show.html.erb:82]: should say 'value'

**Comment by paulmsmith** [comment]
Can tell I got interrupted with a call on this one. Fixed.

**Comment by paulmsmith** [comment]
OK. I've worked out the confusion here. This ticket was for changes I made to: `app/views/funding_application/gp_project/cash_contributions/question.html.erb` so I think you might have also reviewed the cash contributions page itself. :) Anyway I think we're covered now. :)

**Review by ptrelease — APPROVED** [review]

**Pull request merged**